generators/config/templates/default.yml

```diff
@@ -21,9 +21,9 @@
  page_load_idle_timeout: 20
  maximized_window: true # Specify maximized browser window size
 
- # Specify one of the following drivers: selenium, selenium_grid, webkit, poltergeist, phantomjs, sauce,
+ # Specify one of the following drivers: selenium, selenium_grid, webkit, poltergeist, sauce,
  # testingbot, browserstack, crossbrowsertesting, headless_chrome, appium
- driver: phantomjs
+ driver: selenium
```

Comment:
I am not sure about Selenium+Firefox as default for the framework. I would use either headless_chrome or poltergeist